Typeform updates

nickoferrall opened the pull request:
- Added mandatory email field
- Cleaned up the code and removed unused files
- Added a text answer response alongside the multiple-choice answer response

Once we've completed MVP and stretch for the project, we can add:
- Functionality that lets the teacher create questions dynamically
- An analytics page
- Additional Typeform functionality, e.g. add themes, different types of questions (such as yes / no responses), thank you message at the end

dividedsky commented on client/src/components/refreshrsPage/Refreshr.js:
```diff
@@ -78,26 +72,47 @@ function Refreshr(props) {
+          title: questionObject.questionTextOne,
           type: 'multiple_choice',
           properties: {
+            randomize: true,
```
Assuming this randomizes the order of the answers below?

nickoferrall replied:
Yep - exactly. It randomizes all the questions the user will see

nickoferrall commented:
> Looks good, Nick! I tried to check it out but it wasn't running, I think because the paths may have changed? So we might need to update the routes/imports in App.js, or maybe I was just doing something wrong.
> 
> Code looks great, though, nice work! 🔥

Thanks for letting me know! I had removed the Typeform.js file but it was still being called in index and app. I've just removed it and pushed again. Looks like it's now passing the Netlify checks 👍
